src/Xml/XmlParser.cpp

```diff
@@ -604,20 +603,16 @@ const char* XmlDocument::parse(const char* p, void* prevData)
 		return nullptr;
 	}
 
+	location = ParseLocation{};
 	// Note that, for a document, this needs to come before the while space skip,
 	// so that parsing starts from the pointer we are given.
 	if (prevData)
 	{
-		location.first = static_cast<XmlParsingData*>(prevData)->_cursor.first;
-		location.second = static_cast<XmlParsingData*>(prevData)->_cursor.second;
-	}
-	else
-	{
-		location.first = 0;
-		location.second = 0;
```

Comment:
If the default initial values are removed (C++17), this would still need initialization code here.

Reply:
It's fine. The comment was more about potentially contributing changes upstream.

Reply:
Ah.